python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_estimator.py

```diff
@@ -360,7 +362,11 @@ def transform_func(iter, init_param, param):  # type:ignore
             self.state_dict = PyTorchPySparkEstimator._get_state_dict_from_remote(self.model_dir)
             worker_stats = res
         else:
-            self.state_dict = res[0]  # state dicts of all runners would be the same
+            # state dicts of all runners would be the same
```

Comment:
This comment should be modified to: only the state_dict of the rank 0 worker would be returned?